kubectl/kubectl-deploy.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kubectl-deploy-pod
+  namespace: default
+spec:
+  params:
+  - name: manifest
+    description: Content of the resource to deploy
+  - name: image
+    default: gcr.io/cloud-builders/kubectl # it is huge
+    description: Kubectl wrapper image
+  steps:
+  - name: kubeconfig
+    image: $(inputs.params.image)
```

Comment:
This needs to a `$(params.image)` instead :wink:

Reply:
oh, my mistake, thanks 👍